admin/templates/admin/admin.html

```diff
@@ -80,6 +81,33 @@ <h2 class="card-title">Publication Terms</h2>
                         </form>
                     </div>
                 </div>
+
+                <!-- File Formats Section -->
+                <div class="mb-4">
+                    <h2 class="card-title">File Formats</h2>
```

Comment:
Rename to `Preservable file formats` and add a short description. And maybe a link to the file format description in the technical documentation.